pkg/tsdb/jaeger/client.go

```diff
@@ -40,12 +48,12 @@ type ServiceDependency struct {
 	CallCount int    `json:"callCount"`
 }
 
-func New(url string, hc *http.Client, logger log.Logger, traceIdTimeEnabled bool) (JaegerClient, error) {
+func New(hc *http.Client, logger log.Logger, settings backend.DataSourceInstanceSettings) (JaegerClient, error) {
 	client := JaegerClient{
-		logger:             logger,
-		url:                url,
-		httpClient:         hc,
-		traceIdTimeEnabled: traceIdTimeEnabled,
+		logger:     logger,
+		url:        settings.URL,
```

Comment:
If this is coming from settings why do we have it explicitly? Can't this be read as `jaegerClient.settings.url` instead `jeagerClient.url`? 
This is just a curiosity, no need to change anything but just I'm curious.